### ISSUE TITLE
Cannot get blob on vertex

### ISSUE THREAD

**Issue by marc1uk** [AlecAivazis/feynman-node#108]
Would like to have a blob as a vertex. The lines should converge to the center of the blob, but it's not possible - they insist on snapping to the circumference of the blob.

![trash](https://user-images.githubusercontent.com/15327579/96621257-53b17a80-1300-11eb-8c66-8c2cd3ff39e6.png)

**Comment by Spa271985** [comment]
Try adjusting the grid size. That should help to pinpoint the vertex.

**Comment by marc1uk** [comment]
nope, doesn't make a difference. The point is anything you try to connect to a blob snaps to its circumference, whereas i would like the blob to lie directly over the vertex. If you try to place the blob on top of the vertex, the vertex gets moved to again be placed on the circumference.
